src/rich-text-editor.component.js

```diff
@@ -23,7 +23,8 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
       },
       resetEditor() {
         editorRef.current.setHTML('')
-      }
+      },
+      getHTML: serialize
```

Comment:
What is the use case where `props.save` isn't solved, but you need to get the HTML?